pkg/ee/default-application-catalog/applicationdefinitions/kube-vip-app.yaml

```diff
@@ -34,6 +34,13 @@ spec:
           chartVersion: 0.4.4
           url: https://kube-vip.github.io/helm-charts
     version: v0.4.1
+  - template:
+      source:
+        helm:
+          chartName: kube-vip
+          chartVersion: 0.6.5
+          url: https://kube-vip.github.io/helm-charts
+    version: v0.8.4
```

Comment:
(optional) newer version is available https://github.com/kube-vip/helm-charts/blob/main/charts/kube-vip/Chart.yaml